packages/back-end/src/routers/segment/segment.controller.ts

```diff
@@ -185,7 +187,6 @@ type PutSegmentRequest = AuthRequest<
 
 type PutSegmentResponse = {
   status: 200;
-  segment: SegmentDocument;
```

Comment:
Look into why this was removed.

Reply:
When we make a put request to `/segments/${current.id}` we don't do anything with the response.